options/locale/locale_en-US.ini

```diff
@@ -125,6 +125,7 @@ pin = Pin
 unpin = Unpin
 
 artifacts = Artifacts
+are_you_sure = Are you sure?
```

Comment:
I am against adding an `are you sure` message without context.
What would work is `Are you sure you want to delete the artifact '%s'?`.
But that is simply confusing.

Reply:
updated as `confirm_delete_artifact = Are you sure you want to delete the artifact '%s' ?`